cmd/cluster-agent/api/v1/clusterchecks.go

```diff
@@ -149,3 +152,16 @@ func clusterChecksDisabledHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNotFound)
 	w.Write([]byte("Cluster-checks are not enabled"))
 }
+
+// parseClientIP retrieves the http client IP from the remoteAddr attribute
+func parseClientIP(remoteAddr string) string {
```

Comment:
Can we add unit tests to this ? I might be missing something, but if you are using IPV6, would that be supported ?

Reply:
good catch 👌